mozci/console/commands/push.py

```diff
@@ -389,6 +389,174 @@ def _list_tasks(tasks):
                 )
```

Comment:
Given all the movements it's hard to tell: in this file you didn't make logic changes and only refactored the code in separate functions, right?
Could you try running the classify-eval command before/after on the same set of pushes and check if the results are exactly the same to ensure there was no regression introduced?